### DISCUSSION TITLE
Lab 22 Step 6 - Image Error

### DISCUSSION THREAD

**Discussion by IvanA673** [hack4impact-calpoly/starter-pack#12] [Q&A] [answered]
Hey, I was working on my starter pack and I'm currently on the part where we add our first image. I followed the format given but when I try to open my image, I get the error. My website also does not display the image. I tried to use different image types and a different browser to see if that would help. However, I still get the error message. Is there any help I could get on this? 

Also, I tried attaching an image to this discussion, please let me know if they're showing. 

Thank you,
Ivan

![hack4impactQ1](https://github.com/hack4impact-calpoly/starter-pack/assets/68352406/26c9ce92-37ad-48c2-aa73-08bcfc7dd48a)
![hack4impactQ2](https://github.com/hack4impact-calpoly/starter-pack/assets/68352406/b9a91d47-36b6-4c51-a2b6-cad361bf8c62)
![hack4impactQ3](https://github.com/hack4impact-calpoly/starter-pack/assets/68352406/100a4d9c-c678-4a8a-a2fd-b1ad88c6f0be)

**Comment by khoa-l** [comment]
Would you try `./professional.jpeg`? It might be a case of weird path issues.

**Reply by IvanA673** [reply]
I tried that right now, it still has the same issue.

**Reply by khoa-l** [reply, marked as answer]
Hm, I just noticed. Is your image in the same folder as all your other files?

**Reply by IvanA673** [reply]
Yea I do have it in the same folder.

**Reply by IvanA673** [reply]
<img width="599" alt="Screenshot 2023-10-01 164732" src="https://github.com/hack4impact-calpoly/starter-pack/assets/68352406/3cb3a89b-0b1f-4bea-99e8-d99cf724366e">

**Reply by IvanA673** [reply]
Actually, I think that got it to work! Thank you so much
